### PR TITLE
Got error 'Field does not exist: pipeline_stages' in tk show

### PR DESCRIPTION
**Pull request by benjaminhuo**
**What this PR does / why we need it**:
Got error below when tk show:

```shell
tk show environments/loki/
evaluating jsonnet: RUNTIME ERROR: Field does not exist: pipeline_stages
	/root/ben/loki/config/vendor/promtail/scrape_config.libsonnet:6:22-63	object <anonymous>
	/root/ben/loki/config/vendor/ksonnet-util/kausal.libsonnet:409:9-28	builtin function <toString>
	/root/ben/loki/config/vendor/ksonnet-util/kausal.libsonnet:409:7-29	function <anonymous>
	/root/ben/loki/config/vendor/promtail/promtail.libsonnet:38:23-61	object <anonymous>
	During manifestation	
```

### CODE REVIEW

File: production/ksonnet/README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ promtail + {
         }
       ],
       container_root_path: '/var/lib/docker',
+      pipeline_stages: [{
```
Comment (cyriltovena) [inline production/ksonnet/README.md]: weird the default config value has this config.

Comment (cyriltovena) [reply production/ksonnet/README.md]: Why is this happening ? @slim-bean

**Comment by cyriltovena** [comment]
@benjaminhuo could you take a look at why there isn't any default on this ? you should not need to set it by default, but be able to override it.

**Comment by benjaminhuo** [comment]
@cyriltovena something is broken when changing the API version, I'll get the new code and try again.

Close this for now

Thanks

Ben

**Pull request closed without merging**